app/docs/dev.md

```diff
@@ -1,29 +1,207 @@
-# YourFunctionName
-*Brief description of what this evaluation function does, from the developer perspective*
+# buckinghamPiTheorem
+
+Checks that the set of quantities in the response matches the set of quantities in the sense given by the Buckingham Pi theorem.
```

Comment:
Suggested: "Evaluates a candidate list of power products (e.g. `U*L/nu, p/rho/U^2`) for validity as a set of dimensionless quantities that satisfy the Buckingham Pi theorem. The technical details about the theory and its implementation are given in [https://doi.org/10.6084/m9.figshare.24850131.v4](https://doi.org/10.6084/m9.figshare.24850131.v4)"

Reply:
Changed mostly according to suggestion. Added note that the algorithm used in the evaluation function is a modified version of the algorithm in the paper that allows a larger set of expressions.